### PR TITLE
Add Meta Informations to the common Message object

### PR DESCRIPTION
**Pull request by damienalexandre**
See https://github.com/php-translation/loco-adapter/pull/7 and https://github.com/FriendsOfApi/localise.biz/pull/16.

This PR adds Meta informations from the Data Collector Message to the Common Message object. This allow the Loco-Adapter to use this Meta information in order to add the parameters as notes on newly created assets.

### Screenshot

image

### CODE REVIEW

File: Model/SfProfilerMessage.php [hunk 1]
```diff
@@ -116,20 +116,35 @@ public static function create(array $data)
...
+         if (!empty($this->getTransChoiceNumber())) {
+             $meta['transChoiceNumber'] = $this->getTransChoiceNumber();
+         }
+
+         return new Message(
+             $this->key,
+             $this->domain,
+             $this->locale,
+             $this->translation,
+             $meta
```
Comment (rvanlaak) [inline Model/SfProfilerMessage.php]: Seems better to me to add `parameters`, `count` and `transChoiceNumber` to the `Message` value object as fields.

**Comment by damienalexandre** [comment]
Do not merge I still have a cleanup to do:

image

**Comment by Nyholm** [comment]
Great. This looks good. I merged the two downstream PRs and tagged new releases.

File: Model/SfProfilerMessage.php (hunk 1 above)
Comment (damienalexandre) [inline Model/SfProfilerMessage.php]: Maybe, I'm not sure: the Message object is a common objet used to represent a translation on the storage, and those parameters, count and transChoiceNumber information are only needed when displaying a translation.

**Comment by damienalexandre** [comment]
Clean up done, as the profiler was storing all the parameters from all the calls to a translation, the dumped version could become heavy; I reduce the array to only have one occurrence of each parameter key.

**Comment by Nyholm** [comment]
Feel free to drop support for HHVM. Symfony has officially dropped it.

**Comment by damienalexandre** [comment]
HHVM dropped, but there is a PHP7 syntax error now :grimacing: https://travis-ci.org/php-translation/symfony-bundle/jobs/238027526
Can't find it!

**Comment by Nyholm** [comment]
Two things: 

1) Twig fucked up

2) We are installing dev and not releases. We should use: 

```
     - travis_retry composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction		
```

**Comment by damienalexandre** [comment]
Launched the builds again and it's green :+1:

**Pull request merged**